Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -2152,6 +2187,27 @@ theorem isLittleO_pi {ι : Type _} [Fintype ι] {E' : ι → Type _} [∀ i, Nor
   exact ⟨fun h i c hc => h hc i, fun h c hc i => h i hc⟩
 #align asymptotics.is_o_pi Asymptotics.isLittleO_pi
 
+lemma eventuallyLE_of_isLittleO_of_isBigO_symm {f₁ f₂ : α → E'} (hf₁ : f₁ =o[l] g)
+    (hf₂ : g =O[l] f₂) : ∀ᶠ x in l, ‖f₁ x‖ ≤ ‖f₂ x‖ := by
```

Comment:
Thinking back about it, this one is a little bit strange: it can be proved directly with
```
  simpa using IsLittleO.bound (hf₁.trans_isBigO hf₂) zero_lt_one
```
In particular, having the intermediate function `g` feels useless, the real assumption is just that `f_1` is a small o of `f_2`. Maybe put this one just after `isLittleO.def`, specializing to `c = 1`?

Reply:
Good point. I renamed it to `IsLittleO.eventuallyLE` in the process.